app/views/results.nunjucks

```diff
@@ -9,34 +9,84 @@
+      <h2 class="results-none">There are no pharmacies within 20 miles of your location</h2>
+      <p>Call 111 for:</p>
+      <ul>
+        <li>medical advice if you aren’t in an emergency</li>
+        <li>advice on whether you need urgent care</li>
+        <li>more information about a specific health condition</li>
+      </ul>
+      <p>If you need emergency help, call 999.</p>
+    </div>
+  </div>
+
   {% else %}
-    <ol class="list-results">
-      {% if openServices.length > 0 %}
-        <h1 class="local-header--title--question open">Pharmacy nearest to {{ location }} open now</h1>
-        {% for service in openServices %}
-        <li class="list-results-item open" style="background-color:hotpink;">
-          {% include "includes/result-item.nunjucks" %}
+
+    {% if openServices.length > 0 %}
+      <ol class="results">
+        <li class="results__item results__item--nearest">
+          <h2 class="results__type results__type--nearest">Pharmacy nearest to {{ location }} open now</h2>
+          <div class="results__details results__details-nearest column--two-thirds results__details--column">
+            {% set service = openServices[0] %}
+            {% include "includes/result-item.nunjucks" %}
+          </div>
         </li>
-        {% endfor %}
-      {% else %}
-        <p class="all-shut">SOZ, nout open right now</p>
-      {% endif %}
 
-      {% if nearbyServices.length > 0 %}
-        <h1 class="local-header--title--question nearby">Next closest pharmacies to {{ location }}</h1>
+        {% if nearbyServices.length === 0 %}
+
+          </ol>
+          <div class="results">
+            <h2 class="results-none-nearby">There are no pharmacies within 20 miles of your location</h2>
+          </div>
+
+        {% else %}
+          <li class="results__item results__nearby">
+            <h2 class="results__type results__type--nearby">Next closest pharmacies to {{ location }}</h2>
+            <ol class="results__nearby-list">
+            {% for service in nearbyServices %}
+              <li class="results__item results__item--nearby">
+                <div class="results__details results__details--nearby column--two-thirds">
+                {% include "includes/result-item.nunjucks" %}
+                </div>
+              </li>
+            {% endfor %}
+            </ol>
+          </li>
+        </ol>
+        {% endif %}
+
+    {% else %}
+
+      <div class="results">
+
+        <div class="results-block">
+          <h2 class="results-none-open">There are no pharmacies open now within 20 miles of {{ location }}</h2>
+        </div>
+
+        <h2 class="">Next closest pharmacies to {{ location }}</h2>
+        <ol class="results__nearby-list">
         {% for service in nearbyServices %}
-        <li class="list-results-item nearby" style="background-color:cadetblue;">
-          {% include "includes/result-item.nunjucks" %}
-        </li>
+          <li class="results__item results__item--nearby">
```

Comment:
Ignore this comment, I was looking at the two nearby blocks (which could be added into their own include) but due to the open list my original comment doesn't make sense!